Realm/RLMCollection.mm

```diff
@@ -426,11 +428,19 @@ - (void)invalidate {
     }
     bool skipFirst = std::is_same_v<RLMCollection, RLMResults>;
     auto token = [[RLMCancellationToken alloc] init];
+    
+    realm::KeyPathArray keyPathArray;
+    for (NSString *keyPath in keyPaths) {
+        // ???: Is it possible to hit this when a collection doesn't have an RLMClassinfo?
```

Comment:
No, `objectInfo` is nil only for unmanaged collections and we reject those above.